desktop/src/main/fileProcessing/workers/copyWorker.ts

```diff
@@ -182,6 +182,14 @@ const copyDirectoryInBatches = async (
 
     totalFileCount = await countFiles(source);
```

Comment:
No action needed, but better variable names should be `sourceDir` and `destDir`.